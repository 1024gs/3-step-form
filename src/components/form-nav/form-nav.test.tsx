import React from '../../../vendors/react';
import {shallow} from 'enzyme';
import FormNav from './form-nav';

describe('FormNav', () => {
  it('renders', () => {
    const wrapper = shallow(<FormNav />);
    expect(wrapper).toMatchSnapshot();
  });
});
