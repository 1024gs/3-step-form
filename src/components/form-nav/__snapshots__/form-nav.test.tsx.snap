// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormNav renders 1`] = `
<ul
  className="nav nav-pills nav-fill form-nav"
>
  <li
    className="nav-item"
  >
    <Link
      className="nav-link active"
      href="/"
    >
      User
    </Link>
  </li>
  <li
    className="nav-item"
  >
    <Link
      className="nav-link disabled"
      href="/privacy"
    >
      Privacy
    </Link>
  </li>
  <li
    className="nav-item"
  >
    <Link
      className="nav-link disabled"
      href="/done"
    >
      Done
    </Link>
  </li>
</ul>
`;
